openkongqi/source/base.py

```diff
@@ -101,7 +101,7 @@ def get_station_uuid(self, name):
         .. note:: This is not performing any recursive search in the map
         """
         return get_uuid(settings['SOURCES'][self.name]['uuid'],
-                        self._station_map[name])
+                        self._station_map[name]['uuid'])
```

Comment:
This should not be in there, it's part of the other issue...
To remove this from your commit:
```sh
$ git reset --soft HEAD^
$ git status # make sure your files are in staging
$ git reset openkongqi/source/base.py
$ git status # make sure only the base.py file was removed from staging
$ git commit -c ORIG_HEAD
$ git push origin --force  station_wildcard
```